base.scss

```diff
@@ -1,6 +1,8 @@
 @import "./src/styles/scss/reset";
 @import "./src/stories/colors/color-variables";
 
+// the imports below are not advised to be moved around, because
+// they cross-reference various defined variables between each other
```

Comment:
👍 Good comment. I think you should start the sentence with a capital letter. We do that everywhere else.

Reply:
Done 🥇  Thanks for pointing it out.